tests/conftest.py

```diff
@@ -28,9 +28,8 @@ def runner():
 
 
 @pytest.fixture(scope="function")
-def s3_session(aws_credentials):
+def mocked_s3(aws_credentials):
```

Comment:
Just noticed, I'd alphabetize this properly since the name changed

Reply:
Not sure what you mean by this?

Reply:
Sorry, I was working in a different branch with this issue, my bad,  everything is fine!